src/mixins/createProjectorMixin.ts

```diff
@@ -142,6 +143,10 @@ function attach(instance: Projector, { type }: AttachOptions) {
 
 	projectorData.attachPromise = new Promise((resolve, reject) => {
 		projectorData.afterCreate = () => {
+			if (projectorData.originalAfterCreate) {
+				projectorData.originalAfterCreate.call(instance);
```

Comment:
also should the scope be on the instance? the scope should be whatever is bound right? so you should be able to just call it and pass the args as a rest arg?